agc/Makefile

```diff
@@ -0,0 +1,41 @@
+include ../options.mk
+
+CFLAGS+=$(R2_CFLAGS)
+LDLAGS+=$(R2_LDFLAGS)
+
+CFLAGS+=-Wall -shared -fPIC $(LDFLAGS_LIB)
+CFLAGS+=-DLIL_ENDIAN=1
+
+LDFLAGS+=$(LINK) -lr_util
+DESTDIR?=
+
+CURDIR=
+
+TARGET_ASM=asm_agc.$(LIBEXT)
+TARGET_ANAL=anal_agc.$(LIBEXT)
+
+OBJ_ASM=
+OBJ_ASM+=asm.o
+OBJ_ASM+=asm_main.o
+
+OBJ_ANAL=
+OBJ_ANAL+=asm.o
+OBJ_ANAL+=anal.o
+OBJ_ANAL+=anal_main.o
+
+ALL_TARGETS=
```

Comment:
just one line. the list is not that long to justify having muliple concatenations